libnetwork/netavark/config.go

```diff
@@ -232,6 +237,54 @@ func (n *netavarkNetwork) networkCreate(newNetwork *types.Network, defaultNet bo
 	return newNetwork, nil
 }
 
+func createIpvlan(network *types.Network) error {
```

Comment:
this should be shared with createMacVlan() to not duplicate so much code

Reply:
done